index.js

```diff
@@ -1,10 +1,19 @@
-import {NativeModules, DeviceEventEmitter} from 'react-native';
+import {NativeModules, DeviceEventEmitter, Platform, PushNotificationIOS} from 'react-native';
```

Comment:
do we even need to include PushNotificationIOS? most of the functions are implemented